src/server/game/AI/CoreAI/PetAI.h

```diff
@@ -65,6 +67,8 @@ class TC_GAME_API PetAI : public CreatureAI
         void DoAttack(Unit* target, bool chase);
         bool CanAttack(Unit* target);
         void ClearCharmInfoFlags();
+
+        TargetSpellList targetSpellStore;
```

Comment:
if it's used only in a function body scope then it shouldn't be moved to class field